phing/build.yml

```diff
@@ -93,3 +89,5 @@ phpcs:
 project:
   local:
     uri: ${project.local.protocol}://${project.local.hostname}
+  ci:
+    uri: 127.0.0.1:8888
```

Comment:
@grasmash, no problem and your inclination towards breaking this up a bit sounds good to me. Let me know if there is any way that I can assist.

From my perspective, it would be most helpful to have the multisite support during the deployment and `local:sync`/`local:refresh` stages. The lack of strong support here forces me to run many things manually on each of the sites my codebase supports.

Regarding your two comments:
- I agree the `site` namespace could go away. But I do think it adds some clarity around what exactly is transpiring.

- The project `uri` keys are only included as a fallback if there are no site-specific configurations. As I currently have this implemented, these values are immediately used within the `site` namespace (see my code comment), to provide values if none exist. I'm definitely open to changing this, but I didn't see a way to do so without introducing new configuration keys and breaking existing projects. However, any changes this requires could conceivably be addressed with an update hook.